feat : profile page ui

skjaiswal88 opened the pull request.

jishnu-soundspire commented:
Just 2 comments, rest looks good!
Truly apologize for the massive delay in the review 🙇 !

skjaiswal88 commented:
Thanks for the feedback! I’ve updated `next.config.js` to use `remotePatterns` instead of `domains` and changed the subscription image to `/images/placeholder.jpg` in `profile/page.tsx` per your review. Please let me know if everything looks good now!
